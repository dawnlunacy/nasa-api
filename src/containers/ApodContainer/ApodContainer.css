.apod-container {
  height: 100vh;
  width: 100vw;
}

.apod-info {
  align-items: center;
  background-color: rgba(18, 27, 26, 0.918);
  border-radius: 50px;
  display: flex;
  height: 55px;
  justify-content: center;
  width: 80%;
  font-size: 1.5em;
  text-align: center;
  margin: auto;
}

.apod-description {
  justify-self: center;
}