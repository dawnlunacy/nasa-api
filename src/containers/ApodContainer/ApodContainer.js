import React from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import './ApodContainer.css';

export const ApodContainer = ({ apod }) => {

  return (
    <section className="apod-container" style= { apod }>
      <div className="apod-info"> 
        <p className="apod-description"> The background image you see here is the Astronomy Picture of the Day from NASA </p>
      </div>
    </section>
  )
};

export const mapStateToProps = state => ({
  apod: state.apod
});

export default connect(mapStateToProps, null)(ApodContainer);

ApodContainer.propTypes = {
  apod:PropTypes.object
}