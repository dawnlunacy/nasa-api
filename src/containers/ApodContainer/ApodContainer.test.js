import React from 'react';
import { ApodContainer } from './ApodContainer';
import { shallow } from 'enzyme';

describe('ApodContainer', () => {
  let wrapper;

  const mockApod = {}

  beforeEach(() => {
    wrapper = shallow(<ApodContainer 
    apod={ mockApod }
      />)
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
});
