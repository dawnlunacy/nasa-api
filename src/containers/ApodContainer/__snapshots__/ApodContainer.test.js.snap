// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApodContainer should match the snapshot 1`] = `
<section
  className="apod-container"
  style={Object {}}
>
  <div
    className="apod-info"
  >
    <p
      className="apod-description"
    >
       The background image you see here is the Astronomy Picture of the Day from NASA 
    </p>
  </div>
</section>
`;
